packages/admin/src/FilamentManager.php

```diff
@@ -82,6 +84,11 @@ public function registerNavigationItems(array $items): void
         $this->navigationItems = array_merge($this->navigationItems, $items);
     }
 
+    public function registerMeta(array $meta): void
```

Comment:
`registerMeta` sounds weird too - perhaps we'd be better off with a method name like `pushMeta` or `addMeta`, thoughts @danharrin?

Reply:
I like `pushMeta` too. If that's the chosen option I can update and get it pushed.